### DISCUSSION TITLE
Is Quasar v3 going to have a lot of breaking changes?

### DISCUSSION THREAD

**Discussion by Rolanddoda** [quasarframework/quasar#17384] [General - Components / Directives / etc]
My team and I are evaluating Quasar. It looks good so far but we wanted to know what is planned for v3. I found this project [Quasar v3](https://github.com/orgs/quasarframework/projects/2/views/1) but I am not sure if all of them are going to be worked and in general I just want to know what can we expect for v3 in terms of breaking changes.

**Comment by FadhiliNjagi** [comment]
The codease has moved to esm in the beta versions
